### ISSUE TITLE
Questions about deprecation issues

### ISSUE THREAD

**Issue by 2muchexposition** [Huggyrei/SlashBot#5]
I was hoping to install slashbot on my Discord server, but I ran into some changes from the instructions and possible compilation errors. Your instructions are great, but I'm very new to this so I hope it's okay to drop a quick line to see if you have any guidance about how to fix these.
1. I believe this is during NPM install--I got an error that @discordjs/collection@0.7.0 and @discordjs/builders@0.16.0 are no longer supported. I see that those libraries are included in C:\...\Discord\SlashBot-main\node_modules\@discordjs , so should I be aiming to replace the versions in that folder, update a reference elsewhere in the code, or something else?
2. Minor thing, but it seems like bot permissions have changed slightly.  'Read Messages/View Channels' does not seem to exist, so I enabled both “Read Message History” and “View Channels”
3. At some point, I believe during the Node install, I got a “ DeprecationWarning: The `punycode` module is deprecated. Please use a userland alternative instead.” error. I tried googling this, but there seem to be about four different possible fixes, so I’m just wondering if there’s one you recommend.

I know this project is ~3 years old now, so no worries if you don’t have time to address this, but thanks so much either way for creating this bot.

**Comment by Huggyrei** [comment]
Thanks for letting me know. I will take a look and update

**Comment by 2muchexposition** [comment]
That would be amazing, if it's not a huge hassle. Thank you so much!

**Comment by Huggyrei** [comment]
Hi 2muchexposition,
I've put through a few updates including the update for the new Discord api. I've also updated the instructions for setting up and running the bot. Let me know if that's working for you.

**Comment by 2muchexposition** [comment]
Amazing! I'm so excited to try it out, hopefully tonight or tomorrow.

**Comment by 2muchexposition** [comment]
I'm so sorry for the delay--I've been a bit slammed, but I finally got a chance to try this out and it worked almost perfectly. The only blip was that when I originally executed "Node index" I got an error that said "error: new Error("Used disallowed intents")" and then a list of 10 disallowed intents. (I could send to you the specifics that would be helpful.) I read up on that a bit and then tried enabling "Server Members Intent" as well as "Message Content Intent", and that seemed to do the trick. So far I've just tried it by myself, but I'm excited to get a game going in the coming days.
